lib/KalturaFTPServer.js

```diff
@@ -260,13 +297,13 @@ FtpSession.prototype = {
 			else {
 				switch (extension) {
 					case 'xml':
-						This.handleXmlListResponse(listResponse, callback, This, service, currentList, map, pager, currentClient, handler, filter);
+						This.handleXmlListResponse(listResponse, service, currentList, map, pagingHandler);
```

Comment:
since i'm parsing each element differently during the listResponse i need to push it then to the currentList.
I don't see any difference between that and between holding another list of items to push back to the pagingHandler and the pushing it to the currentList.